pkg/syncer/storagepool/service.go

```diff
@@ -121,7 +121,7 @@ func InitStoragePoolService(ctx context.Context,
 			log.Errorf("Creating Kubernetes client failed. Err: %v", err)
 			return
 		}
-		k8sInformerManager := k8s.NewInformer(k8sClient)
+		k8sInformerManager := k8s.NewInformer(ctx, k8sClient, true)
```

Comment:
It should be false here ? based on the PR description. Since this is invoked for Supervisor Cluster only